### PR TITLE
Remove enable-istio flag for e2e test.

### PR DESCRIPTION
**Pull request by cadmuxe**
The test will test if Istio:DestinationRule CRD exist.

### CODE REVIEW

**Comment by cadmuxe** [comment]
@bowei suggested to remove unnecessary flags.

File: pkg/e2e/framework.go [hunk 1]
```diff
@@ -85,15 +86,30 @@ func NewFramework(config *rest.Config, options Options) *Framework {
...
 	}
 	f.statusManager = NewStatusManager(f)
-	if options.EnableIstio {
-		dynamicClient, err := dynamic.NewForConfig(config)
-		if err != nil {
-			klog.Fatalf("Failed to create Dynamic client: %v", err)
-		}
-		destrinationGVR := schema.GroupVersionResource{Group: destinationRuleGroup, Version: destinationRuleAPIVersion, Resource: destinationRulePlural}
-		f.DestinationRuleClient = dynamicClient.Resource(destrinationGVR)
+	apiextensionClient, err := apiextensionsclientset.NewForConfig(config)
```
Comment (freehan) [inline pkg/e2e/framework.go]: Add a comment to describe what you are replacing this with. 

Basically validate if DestinationRule CRD exists and the version is expected.

Comment (cadmuxe) [reply pkg/e2e/framework.go]: Updated, thank you for the review.

**Pull request merged**